src/components/Modal/Modal.vue

```diff
@@ -1,19 +1,22 @@
 <template>
-  <transition name="modal">
-    <div v-if="show" class="fixed w-full h-full left-0 top-0 z-50 flex items-center justify-center bg-black bg-opacity-50 transition-opacity duration-500" @click.self="onClose">
-      <div class="concrete-modal max-w-full bg-white rounded shadow duration-300" :class="widthClass">
+  <transition name="modal" aria-modal="true">
+    <div v-if="show"
+      class="fixed w-full h-full left-0 top-0 z-50 flex items-center justify-center bg-black bg-opacity-50 transition-opacity duration-500 p-4"
+      @click.self="onClose" @keydown.esc="onClose" ref="modal" aria-model="true" role="dialog">
```

Comment:
should this be `aria-modal`?
```suggestion
      @click.self="onClose" @keydown.esc="onClose" ref="modal" aria-modal="true" role="dialog">
```